backend/src/services/admin/admin.service.ts

```
@@ -0,0 +1,191 @@
+    const result = await pool.query(
+      `SELECT id, username, password_hash, created_at, updated_at, created_by, is_root
+             FROM auth.project_admins
+             WHERE username = $1 AND deleted_at IS NULL`,
+      [username]
+    );
+
+    // dummy hash to prevent timing attacks
+    const dummyHash = await bcrypt.hash('dummy', 10);
+
+    if (result.rows.length === 0) {
+      // comparing dummy 
```

ESLint

[error] Delete `·`

(prettier/prettier)